### ISSUE TITLE
[Req]: HTML5 Format / Minify

### ISSUE THREAD

**Issue by dlford** [developertools-tech/developertools.tech#8]
### Explain the proposed feature

HTML5 format / minify tool

**Comment by dlford** [comment]
@ageddesi thanks for the suggestions! Did you want to take a crack at this?

**Comment by ageddesi** [comment]
Could be fun. Will try look over next few days. :)

**Comment by dlford** [comment]
> Could be fun. Will try look over next few days. :)

Awesome! Much appreciated!

**Comment by ageddesi** [comment]
So far I am hitting 'Module not found: Can't resolve 'fs'' issues with the package I am using 
https://github.com/terser/html-minifier-terser
Will need to have another look tomorrow

**Comment by dlford** [comment]
> So far I am hitting 'Module not found: Can't resolve 'fs'' issues with the package I am using https://github.com/terser/html-minifier-terser Will need to have another look tomorrow

Yeah, that's the issue, most of the minifiers are for node, thanks for looking into it for me though!
